New Props by Unclassy

unclassy opened the pull request:
- Added CarSiren, Truck, TruckSiren and Copter
- also a new Skins Folder with base skins and additional ones

unclassy committed:
Add files via upload

unclassy committed:
Add files via upload

Cheesse commented:
Your pull request undoes some of the changes that were made in the last commit. Can you fix your pull request so that doesn't happen? Also, please leave the AdditionalIncludeDirectories and AdditionalLibraryDirectories tags unchanged.

unclassy committed:
Delete .gitignore

unclassy committed:
Delete README.md

unclassy committed:
Delete index.html

unclassy committed:
Delete changelog.txt

unclassy committed:
Delete instructions.txt

unclassy committed:
Delete license.txt

unclassy committed:
Delete index.html

unclassy committed:
Delete cow.png

unclassy committed:
Delete icon.png

unclassy committed:
Delete Minecraftia.ttf

unclassy committed:
Delete index.css

unclassy committed:
Delete instructions.txt

unclassy committed:
Delete MoreModelsPlugin.cs

unclassy committed:
Delete More Models.sln

unclassy committed:
Delete More Models.csproj

unclassy committed:
Delete AssemblyInfo.cs

unclassy committed:
Delete CarModel.cs

unclassy committed:
Delete CaveSpiderModel.cs

unclassy committed:
Delete CowModel.cs

unclassy committed:
Delete CrocModel.cs

unclassy committed:
Delete EndermanModel.cs

unclassy committed:
Delete HuskModel.cs

unclassy committed:
Delete MagmaCubeModel.cs

unclassy committed:
Delete SlimeModel.cs

unclassy committed:
Delete StrayModel.cs

unclassy committed:
Delete VillagerModel.cs

unclassy committed:
Delete WitherSkeleton.cs

unclassy committed:
Delete ZombiePigmanModel.cs

unclassy committed:
deleting unmodified files

Signed-off-by: unclassy <jdumesn@gmail.com>

unclassy commented:
Hi,
I've deleted all the files I didn't modify so you can only need to merge
the updated ones.
I've tried to merge my changes with the ones you made and I've got a
compilation problem with the external references now:
BoxDesc_X, Y and Z Quad2 are not known.
Can you tell me what new reference is missing?

By the way, would it be possible to animate a model without making it walk?
I'm thinking about windmill blades, machinery or other types of static
models that would be nice to animate.

Thanks,

On Sun, Jun 28, 2020 at 4:13 AM Cheesse <notifications@github.com> wrote:

> Your pull request undoes some of the changes that were made in the last
> commit. Can you fix your pull request so that doesn't happen? Also, please
> leave the AdditionalIncludeDirectories and AdditionalLibraryDirectories
> tags unchanged.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/VenkSociety/More-Models/pull/22#issuecomment-650672280>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP3PCRRIDVHY2UNIGZJHP7LRY2RLVANCNFSM4OD33DNQ>
> .
>

UnknownShadow200 commented:
BoxDescX/Y/ZQuad2 were only added 12 days ago, you need to update your version of the client's code and recompile it.

unclassy commented:
I've downloaded and compiled the last client's code.

Now i'm getting 
Error	C2061	syntax error: identifier 'large_vertices'	MoreModels

UnknownShadow200 commented:
`static VertexP3fT2fC4b large_vertices[32 * 32];` needs to be changed to `static struct VertexTextured large_vertices[32 * 32];`

unclassy committed:
Add files via upload

added boat and sailboat, merged changes on original master branch

unclassy committed:
Add files via upload

added boat and sailboat. Updated carsiren

unclassy committed:
Add files via upload

added boat and sailboat

unclassy commented:
Thanks. It worked.

I've added another 2 models (boat and sailboat) and merged your changes on Common.h and MoreModels.c

I've only uploaded the files I've updated/added to avoid any other conflicts specially with the external refererences.

However, you will need to add the new props files to the project.

Is that ok for you?

unclassy committed:
Add files via upload

added plane model

unclassy committed:
Add files via upload

added plane model

unclassy committed:
Add files via upload

added plane model

unclassy committed:
Add files via upload

added plane model, fixed carSiren

unclassy committed:
Add files via upload

added screenshots
